js/tabSwitch.js

```diff
@@ -0,0 +1,54 @@
+import { allTodos, deleteCompletedDiv } from "./todoapp.js";
+import completeTodo from "./completeTodo.js";
+
+const showAll = document.querySelector("#all-button");
+const showActive = document.querySelector("#active-button");
+const showCompleted = document.querySelector("#completed-button");
+
+export default function activeTab() {
+  const tabs = document.querySelectorAll(".nav-section");
+  tabs.forEach((link) => {
+    link.addEventListener("click", () => {
+      tabs.forEach((link) => {
+        link.classList.remove("active");
+      });
+      link.classList.add("active");
+    });
+  });
+}
+
+function todosVisibility(
+  visibility,
+  display1,
```

Comment:
display1 is a display of delete completed div. So name it like `deleteCompletedButtonDisplay`